frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkTypeOptions.tsx

```diff
@@ -112,19 +113,19 @@ const NetworkTypeOptions = (props) => {
                       typeParamsData,
                     )
                   }
-                  checked={_.get(typeParamsData, `${key}.value`, false)}
+                  checked={!_.isBoolean(value) ? parameter?.initValue : value}
```

Comment:
you need to check for `initValue` in all elemnt types (or maybe just init the `value` param and then you don't need this check)